Makefile

```diff
@@ -61,12 +52,83 @@ BUILDFILE_AD_SERVER:=$(BUILDFILE_PREFIX).ad-server
 BUILDFILE_NIGHTLY_AD_SERVER:=$(BUILDFILE_PREFIX).nightly-ad-server
 BUILDFILE_CLIENT:=$(BUILDFILE_PREFIX).client
 BUILDFILE_TOOLBOX:=$(BUILDFILE_PREFIX).toolbox
+OS_PREFIX=$(addsuffix -,$(OS_NAME))
+TAG=$(OS_PREFIX)latest
+NIGHTLY_TAG=$(OS_PREFIX)nightly 
+
+
```

Comment:
Extra new line.

Reply:
Let's not worry too much about extra blank lines. They're hard for @obnoxxx to find visually and they don't matter much in the grand scheme of things. You, synarete, or I can come in another day and clean them up (for example when we add support for different architectures)